cmd/dispenser/server.go

```diff
@@ -71,14 +71,19 @@ const topPageTemplate = `
     <script
       src="https://code.jquery.com/jquery-3.3.1.min.js"
       integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
-      crossorigin="anonymous"></script>
-
-    <script>
+	   crossorigin="anonymous"></script>
+	  
+	 <script>
+	  function loadparm(){
+				const queryString = window.location.search;
+				const urlParams = new URLSearchParams(queryString);
+				$('#target').html(urlParams.get('account'));
+	  }  
       function onload() {
             $('#dispense').click(function (e) {
-              var recaptcha = grecaptcha.getResponse();
-              var target = $('#target').val();
-
+			   var recaptcha = grecaptcha.getResponse();
```

Comment:
nitpick, but please fix the indentation here and below -- the rest of this file uses spaces.

Reply:
OK. This has been fixed. Thanks! Better than new. :)